pkg/image/docker/daemon_provider.go

```diff
@@ -193,7 +205,7 @@ func (p *DaemonImageProvider) Provide() (*image.Image, error) {
 		return nil, fmt.Errorf("unable to save image to tar: %w", err)
 	}
 	if nBytes == 0 {
-		return nil, fmt.Errorf("cannot provide an empty image")
+		return nil, ErrEmptyImage
```

Comment:
is there a specific reason the `ErrEmptyImage` var was introduced?

Reply:
I thought it would be useful to have that error as a var, but no one references that, I will drop it